From cb1aec1b2a291d77aa97c268dec3020ee876eba2 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert+renesas@glider.be>
Date: Wed, 24 Feb 2016 09:13:45 +0100
Subject: [PATCH 1067/1565] drm/amd: Do not make DRM_AMD_ACP default to y

By default, not only this driver is enabled on all platforms, but also
generic PM Domains and Multi-Function Devices.

Drop the "default y" to fix this.

Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/acp/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/acp/Kconfig b/drivers/gpu/drm/amd/acp/Kconfig
index 2b07813..0f734ee 100644
--- a/drivers/gpu/drm/amd/acp/Kconfig
+++ b/drivers/gpu/drm/amd/acp/Kconfig
@@ -2,7 +2,6 @@ menu "ACP Configuration"
 
 config DRM_AMD_ACP
        bool "Enable ACP IP support"
-       default y
        select MFD_CORE
        select PM_GENERIC_DOMAINS if PM
        help
-- 
1.9.1

